File: backend/components/metiers/threed/tests_centralises/guides/helpers/samples/test_usage_samples.js
// test_usage_samples.js – Test ultra avancé pour usage_samples.js (guides.helpers.samples)
const { getUsageSample } = require('./usage_samples');

test('getUsageSample() retourne un objet avec status ok et data', () => {
  const sample = getUsageSample();
  expect(sample.status).toBe('ok');
  expect(sample).toHaveProperty('data');
});

File: backend/components/metiers/threed/tests_centralises/guides/helpers/samples/test_usage_samples.py
"""
test_usage_samples.py – Test ultra avancé pour usage_samples.py (guides.helpers.samples)

Ce fichier fournit un test d’exemple pour usage_samples.py.
"""
from backend.components.metiers.threed.guides.helpers.samples.usage_samples import exemple_usage_samples as get_usage_sample

def test_get_usage_sample():
    sample = get_usage_sample()
    assert sample["status"] == "ok"
    assert "data" in sample
